app/src/main/java/org/oppia/app/customview/interaction/NumericInputInteractionView.kt

```diff
@@ -0,0 +1,20 @@
+package org.oppia.app.customview.interaction
+
+import android.content.Context
+import android.util.AttributeSet
+import android.widget.EditText
+import org.oppia.app.model.InteractionObject
+
+/** The custom EditText class for numeric input interaction view. */
+class NumericInputInteractionView @JvmOverloads constructor(
```

Comment:
Can you add a TODO on #249 here enumerating which attributes we should eventually define in Kotlin/XML that are required for the numeric input interaction to work correctly? At least the following come to mind looking at the test XML below:
- ``android:digits``
- hint
- inputType
- background
- maxLength

Reply:
Resolved